modules/virt-configuring-ip-for-vm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-configuring-ip-for-vm_{context}"]
+= Configuring an IP address for a virtual machine
+
+You can use cloud-init to configure an IP address for a virtual machine. The IP address can be provisioned to be dynamic or static. 
+
+You can add or edit the cloud-init configuration of an existing virtual machine configuration. If you edit a running virtual machine, the virtual machine must be rebooted for the changes to take effect.
+
+.Prerequisites
+
+* DNS is configured via DHCP on the cluster.
+
+* A Linux bridge network attachment definition.
+
+* The virtual machine is connected to the bridge network. 
+
+// web console and CLI? Link to edit in web console?
+
+.Procedure
+
+. Edit or create a virtual machine configuration and include the cloud-init network details in the `spec.volumes.cloudInitNoCloud.networkData` field of the virtual machine configuration:
+
+.. To configure a dynamic IP, specify the interface name and the `dhcp` boolean:
```

Comment:
I would put the DHCP requirement only here. This assumes that a DHCP server must be available on the network.

Reply:
I need to keep the prerequisites together, so I've changed the earlier one to:
"To configure a dynamic IP for the virtual machine, you must have a DHCP server available on the secondary network."